src/Parser/Hunter/Survival/Modules/Abilities.js

```diff
@@ -216,6 +232,30 @@ class Abilities extends CoreAbilities {
           base: 1500,
         },
       },
+      {
+        spell: [SPELLS.CALL_PET_1, SPELLS.CALL_PET_2, SPELLS.CALL_PET_3, SPELLS.CALL_PET_4, SPELLS.CALL_PET_5],
+        category: Abilities.SPELL_CATEGORIES.UTILITY,
+        gcd: {
+          base: 1500,
+        },
+      },
+
+      /**
+       * Following are implemented to avoid annoying console messages.
+       */
+
+      {
+        spell: SPELLS.DEFILED_AUGMENT_RUNE,
```

Comment:
this is already in https://github.com/WoWAnalyzer/WoWAnalyzer/blob/master/src/Parser/Core/CASTS_THAT_ARENT_CASTS.js, does that not prevent the console messages? If not, we should probably make CASTS_THAT_ARENT_CASTS be ignored by the module that is printing the warnings. That way we would also avoid everybody having to make a normalizer just to avoid the console spam. @MartijnHols

Reply:
Nope, they're still in the console if I remove them from abilities.
![image](https://user-images.githubusercontent.com/29204244/43138466-3e14cefc-8f4f-11e8-8c8e-699cdf3c68b2.png)